tests/init/neg/promotion-loop.check

```diff
@@ -1,7 +1,8 @@
 -- Error: tests/init/neg/promotion-loop.scala:16:10 --------------------------------------------------------------------
 16 |  println(b)      // error
    |          ^
-   |          Cannot prove that the value is fully initialized. Only initialized values may be used as arguments.
-   |
-   |          The unsafe promotion may cause the following problem:
-   |          Cannot prove that the value is fully initialized. Only initialized values may be used as arguments.
+   |          Cannot prove the argument is fully initialized.
```

Comment:
1. The arguments of a method call in a constructor must be fully initialized. I could not prove that the underlined argument is fully initialized.
2. The initializer of `b` instantiates `class B`.
3. The constructor of `B` references `test`, the self parameter of class `Test`.
4. The `this` of class `Test` is not yet initialized because the initializer of its field `n` has not yet run.